migrations/2018_01_30_223700_create_user_user_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+return Migration::createTable(
+    'user_user',
```

Comment:
Where does this table come from?

Reply:
It currently isn't used anywhere. It's for user-user relations, like if an extension wanted to allow users to "follow" each other or become friends. But we could potentially remove it from this PR and only add it in when we write the corresponding core code?

Reply:
Yes, please.